tensorflow/lite/micro/tools/make/gcc_download.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+# Called with following arguments:
+# 1 - Path to the downloads folder which is typically
+#     tensorflow/lite/micro/tools/make/downloads
+#
```

Comment:
let's add a function that detects the host_os to bash_helper.sh so that that part gets handled automatically?

I'm hoping that we take this logic into its own bash function:
https://github.com/tensorflow/tensorflow/blob/684632e672f1aa0b0554f482b3459e3cd32216bd/tensorflow/lite/micro/tools/make/bash_helpers.sh#L23-L28